11ty/definitions/adhd.md

```diff
@@ -0,0 +1,30 @@
+---
+title: ADD/ADHD
+slug: adhd
+defined: true
+speech: noun
+reading:
+- text: "What Is ADHD? Meaning, Symptoms & Tests"
+  href: https://www.additudemag.com/what-is-adhd-symptoms-causes-treatments/
+- text: "No Time To Confront Racism In Neurodiversity"
+  href: https://blackgirllostkeys.com/adhd/no-time-to-confront-racism-in-neurodiversity/
+- text: DSM-5 ADHD Diagnosis
+  href: https://www.cdc.gov/ncbddd/adhd/diagnosis.html
+
+---
+
+ADHD is a neurological disorder that affects someone's ability to focus or execute tasks. It is commonly thought of as "the hyper kid in class", though it is much more complex and nuanced.
```

Comment:
```suggestion
attention deficit hyperactivity disorder (initialised to ADHD) is a neurological disorder that affects someone's ability to focus or execute tasks; commonly thought of as "the hyper kid in class", though it is much more complex and nuanced.
```

Reply:
I would also add a note here

Formerly called Attention Deficit Disorder (initialised to ADD).

Reply:
I agree. What do you think about mentioning this in "Appropriate Usage" since technically "ADD" is outdated?